### DISCUSSION TITLE
[Enhancement]: Ability to view all Loved albums in same place

### DISCUSSION THREAD

**Discussion by squintgit** [ciderapp/Cider#793] [Feature Request]
### The feature you would like

The iOS app "Marvis Pro" has a way to view all your Loved albums in one place. I find it very useful for keeping track of all my favorite albums, which is what music should be all about IMO. Artists make albums, and we should find our favorites and really get into them. Cider should make it really easy to do that.

### Alternatives and Other Information

Marvis Pro: https://apps.apple.com/us/app/marvis-pro/id1447768809

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by squintgit** [comment]
The "Library" section in the side bar could have a "Loved Albums" and maybe even a "Loved Songs" section.

**Comment by KillerParticles** [comment]
All feature requests now belong in the discussions tab under Feature Request.

**Comment by squintgit** [comment]
Should probably update the app to direct them there. I just went (in the macOS app) to Support > Report a > Feature Request.

**Comment by KillerParticles** [comment]
Yeah its a recent change.
